catalog/task/push-sbom-to-pyxis/0.1/push-sbom-to-pyxis.yaml

```diff
@@ -0,0 +1,41 @@
+---
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: push-sbom-to-pyxis
+  labels:
+    app.kubernetes.io/version: "0.1"
+  annotations:
+    tekton.dev/pipelines.minVersion: "0.12.1"
+    tekton.dev/tags: release
+spec:
+  description: >-
+    Tekton task that extracts sbom from a pull spec and pushes it to Pyxis.
+  params:
+    - name: imageURL
+      description: URL of the main binary image.
+      type: string
+  volumes:
+    - name: workdir
+      emptyDir: {}
+  steps:
+    - name: download-sbom-for-image
+      # Image taken from https://github.com/sigstore/cosign/pkgs/container/cosign%2Fcosign
+      image:
+        ghcr.io/sigstore/cosign/cosign:03468a1498876ccfb72e06f63fdb4f1c562dbca1
```

Comment:
OK, waiting for Scott's input. But otherwise I'd say if it's good enough for the Build team shouldn't it be good enough for us as well?

Reply:
"If it's good for the Build and for the Release it would be good for a third one."

I think this is the kind of things that should be addressed earlier. If that image gets removed by an external entity, the whole release process is compromised.

Reply:
OK, I will ping Scott today about this. We'll need to pick one of the options:

1. Rely on the ghcr.io image - probably not
2. Install cosign in our release-utils image and use that
3. Copy the cosign image manually to Quay and use that

Reply:
Michal copied the image to Quay and they will also use it here: https://github.com/redhat-appstudio/build-definitions/pull/288/files

I updated the task to use the image from Quay.

Reply:
We discussed this on yesterday's standup call and both Scott and David agreed with this solution. Resolving.